source/about.rst

```diff
@@ -3,57 +3,119 @@
 About
 #####
 
-The |CL| delivers a secure, hardware optimized OS. Its easy updates ensure that
-software dependencies remain mutually compatible.
-
-|CL| does this via custom infrastructure components and process innovations.
+|CL-ATTR| does things differently. Our software architecture provides a 
+unique and innovative platform for Linux* developers focused on
+performance, security, and cutting-edge computation in the cloud. 
 
 .. contents::
    :local:
    :depth: 1
 
 
-For detailed information on these topics, refer to the :ref:`cl-guides` guides.
+What is |CL|?
+*************
+
+|CL| is an open source, rolling-release Linux* distribution, optimized for performance and security from the cloud to the Edge. Designed from the ground up, |CL| provides an industry blueprint on how to incorporate Intel® architecture features for a modern, modular Linux* OS. |CL| is not based on any other Linux distro. 
+
+What |CL| isn't?
+****************
+
+|CL| is not intended to be a general-purpose Linux distribution, suitable 
+for novice end-users. While we ship common applications, our purpose isn’t 
+to make an OS for routine desktop tasks and provide immunity from all 
+security threats in all situations. Our unique focus means what we consider 
+*essential* use cases, *optional* use cases, or even *unwanted* use cases, differs from other Linux distros. See our :ref:`target audience <target-audience>` below. 
+
+Is |CL| completely Open Source?
+*******************************
+
+Wherever possible, |CL| aims to be completely open source.  Our 
+`source code`_ is available on GitHub\*. When considering projects for inclusion, we check that they are in active development and are well maintained. We have a very strict requirement for not accepting proprietary packages and non-open source components.  For example, many Linux distros may not be able to include certain media codecs due to  
+:ref:`licensing restrictions <licensing_restrict>`, but `alternatives`_ are available. 
+
+.. _target-audience:
+
+Who is the target audience?
+***************************
+
+|CL| mainly targets professionals in IT, DevOps, Cloud/Container deployments, and :abbr:`AI (Artificial Intelligence)`. 
+
+Rather than making a standard Linux distribution, the |CL| team decided to
```

Comment:
Right now this sentence doesn't make much sense. It says "rather than make our own linux, we decided to make our own linux". Instead, it should say something like "Rather than making a standard X, the team decided to make a special type of X".

Reply:
That's not entirely accurate. The sentence does state, "Rather than making a *standard* Linux distribution...". However, I will revise "our own" to a "special type of X," as you suggest.